content/en/_index.html

```diff
@@ -40,6 +40,22 @@ <h3 class="section-head">
   </div>
 </div>
 
+<div id="overview" class="text-center">
```

Comment:
We probably want to use `id="adopters"` so we can do custom styles here. Not sure if this will disable some styles you are expecting that are coming from `overview`, but if so, copy them.